support-frontend/assets/pages/supporter-plus-landing/components/threeTierTsAndCs.tsx

```diff
@@ -114,7 +115,8 @@ export function OfferTsAndCs({
 		<div css={container}>
 			<p>
 				{' '}
-				<sup style={{ fontWeight: 'lighter', fontSize: '14px' }}>**</sup> Free
+				{/*check this UI*/}
```

Comment:
has this been checked now?

Reply:
I need to go through this and check for these comments as I just spotted some.

Reply:
It's not actually being used at the moment so I think it is fine to make this change